src/common/home/sections/banner.jsx

```diff
@@ -0,0 +1,16 @@
+import { lazy, Suspense } from "react";
+import "./banner.css";
+
+const LazyBanner = lazy(() =>
+  true
+    ? import("common/activities/hackathon/HackathonBanner")
+    : import("common/defaultBanner/DefaultBanner")
+);
+
+export default function Banner() {
+  return (
+    <Suspense fallback={<div className="home_banner_fallback" />}>
```

Comment:
any specific reason for writing className in this format
everywhere it's written in kebab case, keep it consistent